contracts/ibc_transfer/src/contract.rs

```diff
@@ -183,11 +185,21 @@ fn execute_set_fees(
     ack_fee: u128,
     timeout_fee: u128,
     denom: String,
+    payer: Option<String>,
 ) -> StdResult<Response<NeutronMsg>> {
+    let payer = match payer {
+        Some(payer) => match deps.api.addr_validate(&payer) {
+            Ok(addr) => Some(addr),
+            Err(_) => None,
```

Comment:
I think if we don't return `Err` here it kinda incorrect, since if payer is invalid, we may not intent to pay from contract. Like you did in other `execute_set_fees` method?